src/components/common/ErrorBox.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { func, object } from 'prop-types';
+import { connect } from 'react-redux';
+import { clearErrors } from 'actions/commonActions';
+import {
+  injectIntl,
+  defineMessages,
+  intlShape
+} from 'react-intl';
+
+const ErrorBox = (props) => {
+  const { errors, intl } = props;
+
+  const closeErrorBox = (error) => {
```

Comment:
I think there's no need for this extra function and you can use `clearErrors` from the props in the render function right-away.